gwu-project-1/script.js

```diff
@@ -0,0 +1,15 @@
+let inputText = document.querySelector("#input-text");
+let addBtn = document.querySelector("#add-btn");
+let ol = document.querySelector("ol");
+
+let addName = () => {
+    let name = inputText.value
+    let li = document.createElement("li")
+    
+    li.innerText = name
```

Comment:
Your logic doesn't account for the edge case of when the input field is empty because when the field is empty, it keeps adding the empty values to the list as shown in the image below:

![image](https://user-images.githubusercontent.com/125711534/225493227-908751e4-c3ea-4558-b583-21f0bc569e67.png)

Reply:
![git screencshot 01](https://user-images.githubusercontent.com/92020171/225738940-79aafc27-49f4-4e5c-b609-0ba83a63bfca.png)

I have made the adjustment